### PR TITLE
[INTPROD-3645] Catch 404 on PR Update

### PR DESCRIPTION
**Pull request by colinsl**
github.GithubException.UnknownObjectException: 404

### CODE REVIEW

**Comment by colinsl** [comment]
PTAL @haoxue1108 @juliahorowitz

**Comment by haoxue1108** [comment]
👍

**Comment by colinsl** [comment]
PTAL @ryan-lane

File: osscla/webhook_worker.py [hunk 1]
```diff
@@ -60,7 +60,10 @@ def handle_webhook(message):
     if data['action'] == 'closed':
         logger.debug('Skipping closed action')
         return
-    gh.update_pr_status(data['full_repo_name'], data['pr_number'])
+    try:
+        gh.update_pr_status(data['full_repo_name'], data['pr_number'])
+    except Exception:
+        logger.exception(f"Failed to update status for {data['full_repo_name']}#{data['pr_number']}")
```
Comment (dschaller) [inline osscla/webhook_worker.py]: I'm pretty sure that this has python 2 support at the moment. f-strings won't work in those environments

Comment (colinsl) [reply osscla/webhook_worker.py]: We should already be on Py3 from this PR https://github.com/lyft/osscla-private/pull/285

Comment (dschaller) [reply osscla/webhook_worker.py]: This is a public repo used by others. The pr you linked is a private repo.

**Pull request merged**